.github/workflows/lint.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
 
             - name: Check if any dependencies are compromised
               if: always() && steps.install.outcome == 'success'
-              run: /home/runner/.symfony/bin/symfony check:security
+              run: /home/runner/.symfony5/bin/symfony check:security
```

Comment:
Hm, but Symfony Demo is on Symfony 5 already 🤔

Reply:
this is not about the version of Symfony, but the installation path of the CLI tool